example/lib/main.dart

```diff
@@ -47,12 +47,22 @@ class _DemoState extends State<Demo> {
 
   int pageIndex = 0;
 
+  final buttonsFocus = FocusNode();
+  final indicatorsFocus = FocusNode();
+  final fieldsFocus = FocusNode();
+
   final List<Widget> pages = [
     ButtonsPage(),
     IndicatorsPage(),
     FieldsPage(),
   ];
 
+  Color textLuminance(Color backgroundColor) {
+    return backgroundColor.computeLuminance() > 0.5
```

Comment:
`>=`

Reply:
Why

Reply:
@bdlukaa

Reply:
Because the light color should be from 0 to 0.49, and the dark color should be from 0.5 to 1.0. It's used this way all around the flutter framework